hack/ocp-install-nightly-art-operators.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# This script is an automation of steps 1-7 from
+# https://hackmd.io/0vbbJdZ7RXem69vCWI61ug to install the nightly ART bundle
+# build of the OpenShift operators.  The script requires the user to have a
+# employee-token created at least once before
+# (https://source.redhat.com/groups/public/teamnado/wiki/brew_registry).
+
+set -e
+
+function wait_for_all_nodes_ready() {
+  i=20
+  while [ $i -gt 0 ]; do
+    if [[ $(oc get no | grep SchedulingDisabled) ]]; then
+      # reset timer
+      i=20
+      echo "found node with SchedulingDisabled. Resetting timer..."
+    else
+      echo "Waiting ${i} times to ensure all nodes are ready..."
+      i=$((i - 1))
+    fi
+
+    sleep 1
+  done
+}	
+
+brew_credentials=$(curl --negotiate -u : https://employee-token-manager.registry.redhat.com/v1/tokens -s) #make sure you created a token before (https://source.redhat.com/groups/public/teamnado/wiki/brew_registry)
```

Comment:
how about to test if command fails and show the message:

"#make sure you created a token before (https://source.redhat.com/groups/public/teamnado/wiki/brew_registry)" 
exit 1